gammapy/datasets/map.py

```diff
@@ -1128,6 +1134,59 @@ def from_dict(cls, data, lazy=False, cache=True):
         dataset = cls.read(filename, name=data["name"], lazy=lazy, cache=cache)
         return dataset
 
+    @classmethod
+    def from_images(cls, images, energy_min, energy_max, name=None, **kwargs):
+        """Create a MapDataset from a set 2D images which do not have an
+        energy axis. An appropriate energy axis is added for all the images.
+
+        This can be useful for modelling and fitting data reduced outside gammapy
+
+        Parameters
+        ----------
+        images: dict
```

Comment:
Maybe the documentation and usage becomes simpler (especially for the `PSFKernel`  case) if the maps are passed directly like: `.from_images(counts, background, exposure, psf_kernel,  )`. I think there is also some documentation missing...